### PR TITLE
#166151265 Muscle List Could be Updated

### PR DESCRIPTION
**Pull request by willeswa**
#### Description
Update muscle list with an additional list of muscles
#### Type of change
- [x] Currently, there are only 16 muscles in the muscle list. These can be viewed from the muscle .svg image on the muscle overview page or via a rest client.
- [x] This feature will add three more muscles to this list
#### How Has This Been Tested?
- Clone [this repo](https://github.com/andela/wg-baqi.git) to your local computer
- CD into `wger` and create a virtual env by doing `python3 -m virtualenv venv`
- Activate the virtual env: `source venv/bin/activate` && install requirements `pip install -r requirements.txt`
- Run migrations by invoking the `bootstrap-wger` command as is on the Readme.rst
- In your browser, navigate to `http://127.0.0.1:8000/en/exercise/muscle/overview/`
- Compare the diagram with the [original wger website](https://wger.de/en/exercise/muscle/overview/)
- You should see more muscles
#### Any background context:
None
#### Story ID
[#166151265](https://www.pivotaltracker.com/story/show/166151265)
#### Screenshots
Screen Shot 2019-06-06 at 21 29 43
Screen Shot 2019-06-06 at 21 29 52

### CODE REVIEW

File: wger/exercises/fixtures/muscles.json [hunk 1]
```diff
@@ -126,5 +126,29 @@
...
+            "is_front": true,
+            "name": "Forearms"
+        }
+    },
+    {
+        "pk": 19,
+        "model": "exercises.muscle",
+        "fields": {
+            "is_front": false,
+            "name": "Posterious Deltoid"
```
Comment (wekesa931) [inline wger/exercises/fixtures/muscles.json:151]: I am not able to see this muscle `Posterious Deltoid` in the diagram. Was it added?

File: wger/exercises/fixtures/muscles.json [hunk 2]
```diff
@@ -126,5 +126,29 @@
             "is_front": false, 
             "name": "Triceps brachii"
         }
+    },
+    {
+        "pk": 17,
+        "model": "exercises.muscle",
+        "fields": {
+            "is_front": false,
+            "name": "Lateral Deltoid"
```
Comment (wekesa931) [inline wger/exercises/fixtures/muscles.json:135]: This muscle, `Lateral Deltoid` is not present in the diagram.

**Review by wekesa931 — COMMENTED** [review]
Do you think a test can be added to show the muscle list has increased?

File: wger/exercises/fixtures/muscles.json (hunk 1 above)
Comment (willeswa) [reply wger/exercises/fixtures/muscles.json:151]: Apparently, I had not added the muscles. You should now be able to see them.

File: wger/exercises/fixtures/muscles.json (hunk 2 above)
Comment (willeswa) [reply wger/exercises/fixtures/muscles.json:135]: Fixed in the recent push. Kindly see the previous comment.

**Review by wangonya — COMMENTED** [review]
Nice touch on adding the `.svg`s 👍🏾

**Review by jmusila — APPROVED** [review]
Great job

**Review by ceexon — APPROVED** [review]

**Comment by caveinn** [comment]
Good work @willeswa, LGTM

**Pull request merged**